packages/plugins/yggdrasil-worker-forwarder/yggdrasil-worker-forwarder.spec

```diff
@@ -0,0 +1,57 @@
+%define debug_package %{nil}
+
+%if ! 0%{?gobuild:1}
+%define gobuild(o:) GO111MODULE=off go build -buildmode pie -compiler gc -tags="rpm_crashtraceback ${BUILDTAGS:-}" -ldflags "${LDFLAGS:-} -linkmode=external -B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \\n') -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '" -a -v %{?**};
+%endif
+
```

Comment:
Considering the discussion below about `go-srpm-macros` I believe this is redundant
```suggestion
```

Reply:
`gobuild` is not defined in EPEL7's `go-srpm-macros`, while it is in CentOS8 ones.